lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -181,15 +182,28 @@ div.course-wrapper {
         border-top: 4px solid #22B557 !important;
         margin: 0 auto;
       }
+      &.success-left-bar {
+        border-left: 4px solid #22B557 !important;
+        margin: 0 auto;
+      }
       &.message-top-bar {
         border-top: 4px solid #FAB95C !important;
         margin: 0 auto;
       }
+      &.message-left-bar {
+        border-left: 4px solid #FAB95C !important;
+        margin: 0 auto;
+      }
       &.failure {
         border-left: 4px solid #C93B34 !important;
         margin: 0 auto;
       }
     }
+    div.entrance {
+      button.gated-sequence {
+        padding: 0px 100px 0px 50px;
```

Comment:
`opx` is not necassary `o` will do just fine  and use `$baseline` variable like ($baseline*5) equals to `100px` and ($baseline*2.5) which is equal to `50px`